install.sh

```diff
@@ -485,33 +490,45 @@ configure_linux_token() {
 postinstall_check() {
   detect_mondoo
   if [ $MONDOO_INSTALLED = false ]; then
-    red "${MONDOO_PRODUCT_NAME} installation failed (can't find the Mondoo binary)."
+    red "${MONDOO_PRODUCT_NAME} installation failed (can't find the ${MONDOO_BIN} binary)."
     exit 1
   fi
 
   echo "${MONDOO_PRODUCT_NAME} installation completed."
 }
 
 finalize_setup() {
+
+  # If registration token is provided, register client
   configure_token
 
   # Display final message
-  detect_mondoo_registered
-  if [ $MONDOO_IS_REGISTERED = false ]; then
-    purple_bold "\n${MONDOO_PRODUCT_NAME} is ready to go!"
-    echo
-    lightblue_bold "Next you should register ${MONDOO_PRODUCT_NAME} to get access to policies and reports."
-    lightblue_bold "Follow this guide: "
+  purple_bold "\n${MONDOO_PRODUCT_NAME} is ready to go!"
+
+  # Only if installing Mondoo Client, warn user to register. Do not warn open source users.
+  if [ "$MONDOO_PRODUCT_NAME" = "Mondoo Client" ]; then
```

Comment:
I think I'm missing something, under what circumstance would $MONDOO_PRODUCT_NAME now equal "Mondoo Client"?  

Also, it's better to use == here.

Reply:
Never mind about the ==, it's more correct but applied inconsistently elsewhere in the script.

Reply:
Neither of the above are blockers, I'm approving.

Reply:
I was looking for the `=` vs `==` thing, and stackoverflow said that "`==` is a bash-specific alias for `=`," and that using `==` is bad form because it's not POSIX.

It's on stackoverflow! It must be true!

https://stackoverflow.com/questions/20449543/shell-equality-operators-eq

Reply:
`finalize_setup` is always called on line 575, regardless of the value of `$MONDOO_PRODUCT_NAME`.